feature-wallet-impl/src/main/java/jp/co/soramitsu/wallet/impl/presentation/cross_chain/setup/CrossChainSetupViewModel.kt

```diff
@@ -213,23 +214,41 @@ class CrossChainSetupViewModel @Inject constructor(
         }
     }.stateIn(viewModelScope, SharingStarted.Eagerly, defaultAmountInputState)
 
+    private val hasDestinationFeeAmountFlow = MutableStateFlow(false)
```

Comment:
Do we really need this field? Can we use the null state of destinationFeeAmountFlow as hasDestinationFeeAmountFlow.value = false.

Reply:
Yes, we need this field, because when fees are loading inside `destinationFeeAmountFlow`, `destinationFeeAmountFlow` has non-null value at this moment, but we need to show user loading state and don't let user go to the next screen while fees are loading

Reply:
We have LoadingState for cases like this